src/fetch.py

```diff
@@ -7,8 +7,8 @@ def fetch(url):
         response = requests.get(url)
         if response.status_code == 200:
             return response.json()
-        else:
-            raise RuntimeError(f"Request failed {response.status_code}")
+        
+        raise RuntimeError(f"Request failed {response.status_code}")
     except requests.exceptions.RequestException as e:
         """log error message"""
```

Comment:
Why do you need raise exception when you already have try catch block?

Reply:
yes there is no need  will correct it